### DISCUSSION TITLE
Fieldset conditionals

### DISCUSSION THREAD

**Discussion by ottz0** [statamic/cms#9836] [General]
I'd like to conditionally show or hide field-sets in blueprints or in other linked field-sets.  Is it possible to build this feature? I can only show or hide conditionally if its a part of the bluperint and not a fieldset.

Also I'd like to add a section and then conditionally show or hide the whole section, however the section component is only for UI, So I need to show and hide every single item within that section

**Comment by duncanmcclean** [comment]
> I'd like to conditionally show or hide field-sets in blueprints or in other linked field-sets. Is it possible to build this feature? I can only show or hide conditionally if its a part of the bluperint and not a fieldset.

Sorry, I'm not quite sure I understand what you're wanting here. Would you be able to expand?

> Also I'd like to add a section and then conditionally show or hide the whole section, however the section component is only for UI, So I need to show and hide every single item within that section

You could use the [Group fieldtype](https://statamic.dev/fieldtypes/group) for this. You'd only need to add the conditional to the Group field, not any of the fields inside it.

**Reply by jensolafkoch** [reply]
I have the exact same problem. When I use a fieldset in a blueprint, there is no option to apply the usual conditional show or hide logic. The same is true for sections.

To use a group field to wrap a fieldset or a field adds unnecessary complexity and somewhat thwarts the purpose of fieldsets to make using fields easier, and it prevents fields/fieldsets from being moved around easily individually because they are stuck in a group. (Group fields also add superfluous labels on the blueprint, especially if used for just one field or a fieldset with just one field.)

If it were possible to give them both (fieldsets and sections) access to the conditional view/hide logic, that would be a big plus in terms of usability!

I'm not sure if it's technical feasible, but wouldn't it be better if we had an option _in the steering/controlling field_ to decide which other fields (identified by their handles) should be shown or hidden?  By way of a list of handles?

For example: If you have a toggle, offer a select option there to create a list of handles of other fields, fieldsets and sections which should be conditionally hidden. 

That would have the added benefit that it's no longer necessary to duplicate the same kind of logic in a bunch of fields (and no, putting them in groups is not always an option if they are sprinkled throughout the blueprint.)

**Reply by robdekort** [reply]
https://github.com/statamic/ideas/issues/451
